charters/UNITY/charter.md

```diff
@@ -0,0 +1,56 @@
+# Unity Benchmarking Service
+
+## Description
+
+The Unity benchmarking service automates the process of running and benchmarking pipelines.
+
+## Objectives
+
+The Unity benchmarking service aims to automate benchmarking for HCA pipelines, enforce standards for consistent reporting for historic comparison and enable the public to participate in suggesting improvements to pipelines by empowering them to implement improvements and test results using the same infrastructure as the HCA data processing pipelines team. The Unity benchmarking service will be used by those making pipelines for the HCA data processing service and will be public, so anyone can benchmark their implemented ideas against HCA pipelines.
+
+## In-scope
+
+* Benchmarking for HCA pipelines will be automated and included in Unity
+* Benchmarking data will be available through Unity but not controllable by the users
+* Benchmarking data can change without notice but will be consistent in format and schema with previous data or notice will be given to users
+* Leaderboards showing how pipelines differ will be created as different pipelines for the same assay are collected
+* Documentation on how to use Unity and the assumptions pipelines must meet to be ran using Unity
+* Documentation on benchmarking process and outputs
+
+### Scientific "guardrails"
+
+* Benchmarking pipelines are reviewed by the HCA project and treated as data processing pipelines (engineering updates do not require HCA oversight; scientific updates do).
```

Comment:
I think this statement is overly cautious to the point of being a bit confusing. 
- Benchmarking pipelines, as I understand them, are either (1) glue to test or summarize the outputs of our gold standard or community pipelines, or (2) the gold standard pipelines themselves. 

In either case, there is no new scientific product here outside what is covered in the next line. I would remove this line, and might add a definition for what a benchmarking pipeline is.

Reply:
It is definitely within the purview of the AWG to be aware of changes to benchmarking pipelines. It is our shared contract of how we interpret pipeline performance and is more than benchmarking cost and speed but includes judgment on scientific insight (how to use metrics and what metrics to use). Benchmarking pipelines in UNITY will guide the growth and development of the pipelines and affect their science.

Reply:
I will try to clarify.